exporters/jaeger-thrift/src/main/java/io/opentelemetry/exporter/jaeger_thrift/Adapter.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package io.opentelemetry.exporters.jaeger;
+package io.opentelemetry.exporter.jaeger_thrift;
```

Comment:
Hm. I don't like underscores in package names. maybe `io.opentelemetry.exporter.jaeger.thrift`?

Reply:
Cannot do that I think, because of the java9 modules, the grpc is in jaeger so I think we cause a module conflict.

Reply:
I can move the grpc one to `jaeger.grpc` and this to `jaeger.thrift` that should be safe

Reply:
Also fyi the checkstyle failed :)) cannot have "_" in the name :)). I used for the moment `jaeger.thrift` but can someone confirm if that is a problem or not with java9 modules?

Reply:
I believe that JPMS does not restrict sub-packages from being in a separate module, as long as each package is only exported by a single module. There's really no such thing as a "sub package" in java. There shouldn't be any issues. But, also, having `.grpc` for the other one sounds totally fine to me, as well.